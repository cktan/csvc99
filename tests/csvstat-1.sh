# Test Case : Basic Functionality 
../csvstat in/csvstat-1.csv 
