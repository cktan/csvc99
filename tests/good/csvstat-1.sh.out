      #bytes: 37
       #rows: 2
    #columns: 3
avg row size: 18
min row size: 17
max row size: 20
